api/test/integration/conftest.py

```diff
@@ -12,7 +12,9 @@
 import urllib3
 import yaml
 from py.xml import html
+from git.repo import Repo
 
+global env_mode
```

Comment:
The `env_mode` variable is initiated below and set in `api_test()` so it's not necessary to declare it here

```suggestion
```